R/add_coordinates.R

```diff
@@ -25,9 +25,11 @@
 #' ef <- add_coordinates(Brazil_epiflows, YF_coordinates[-1])
 #' get_coordinates(ef)
 #' get_coordinates(ef, location = "Espirito Santo") # coordinates for Espirito Santo
-#' \donttest{
+#' \dontrun{
```

Comment:
```suggestion
#' if (interactive()) {
```

I would use `if (interactive())` to make sure these tests get skipped. CRAN has an inconsistent policy regarding `\dontrun` and `\donttest` and I've moved to just never using them in my examples. `if (interactive())` makes sure that they never run on R CMD check

Reply:
Good point! Alternatively, I though about... commenting these lines out altogether :-)

Reply:
@zkamvar I'm sorry, I made the change myself instead of applying your suggestion...